frontend/talentsearch/src/js/components/pool/PoolApplicationThanksPage.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { useIntl } from "react-intl";
+import { getLocale } from "@common/helpers/localize";
+import { commonMessages } from "@common/messages";
+import Breadcrumbs from "@common/components/Breadcrumbs";
+import type { BreadcrumbsProps } from "@common/components/Breadcrumbs";
+import Link from "@common/components/Link";
+import { useDirectIntakeRoutes } from "../../directIntakeRoutes";
+import { useGetPoolQuery } from "../../api/generated";
+import type { Pool } from "../../api/generated";
+
+interface PoolApplicationThanksProps {
+  pool: Pool;
+}
+
+const PoolApplicationThanks: React.FC<PoolApplicationThanksProps> = ({
+  pool,
+}) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+  const paths = useDirectIntakeRoutes();
+
+  const links = [
+    {
+      title: intl.formatMessage({
+        defaultMessage: "Pools Index",
+        description: "Breadcrumb title for the pools index link.",
+      }),
+      href: paths.home(),
+    },
+
+    {
+      title: pool?.name?.[locale],
+      href: pool ? paths.pool(pool.id) : undefined,
+    },
+
+    {
+      title: intl.formatMessage({
+        defaultMessage: "Apply",
+        description: "Breadcrumb title for the pool application link.",
+      }),
+      href: pool ? paths.poolApply(pool.id) : undefined,
+    },
+
+    {
+      title: intl.formatMessage({
+        defaultMessage: "Thanks for Applying",
+        description:
+          "Breadcrumb title for the pools 'thanks for applying page' link.",
+      }),
+    },
+  ] as BreadcrumbsProps["links"];
+
+  return (
+    <>
+      <Breadcrumbs links={links} />
+      <h1>
+        {intl.formatMessage({
+          defaultMessage: "Thanks for applying to",
+          description: "Title for page thanking user for applying to a pool",
+        })}{" "}
+        {pool.name?.[locale]}
```

Comment:
Can we use the `react-intl` message syntax here to inject the pool name into message for translators?

```suggestion
        {intl.formatMessage({
          defaultMessage: "Thanks for applying to {poolName}",
          description: "Title for page thanking user for applying to a pool",
        }, { 
          poolName: pool.name?.[locale]
        })}
```